code/modules/events/spacevine.dm

```diff
@@ -1,7 +1,7 @@
 /datum/round_event_control/spacevine
 	name = "Spacevine"
 	typepath = /datum/round_event/spacevine
-	weight = 15
+	average_time = 40

```

Comment:
Vines, on the other hand, are not nearly as deadly as aliens, and thus should be lower.